src/basics.md

```diff
@@ -1343,6 +1344,47 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-datetime-arithmetic]: #ex-datetime-arithmetic
+<a name="ex-datetime-arithmetic"></a>
+## Add or subtract dates and times using checked arithmetic
```

Comment:
I'd suggest renaming the example to "Perform checked date and time calculations". It will be slightly more discoverable.